crypto/src/main/java/org/springframework/security/crypto/bcrypt/BCrypt.java

```diff
@@ -13,252 +13,363 @@
 // OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 package org.springframework.security.crypto.bcrypt;
 
-import java.io.ByteArrayOutputStream;
```

Comment:
Please remove unnecessary formatting changes in here so it is easier to follow what has changed.